content/blog/2024/08/26/2024-08-26-gsoc-manage-github-permissions.adoc

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: GSOC Manage jenkinsci GitHub permissions as code
+tags:
+- gsoc
+- gsoc2024
+authors:
+- Alaurant
+opengraph:
+  image: /images/gsoc/opengraph.png
+sig:
+- gsoc
+---
+This blog showcases all the work done in the project link:https://www.jenkins.io/projects/gsoc/2024/projects/automating-rpu-for-jenkinsci-organization/[Manage jenkinsci GitHub permissions as code] during Google Summer of Code 2024.
```

Comment:
```suggestion
= GSOC Manage jenkinsci GitHub permissions as code
	
This blog showcases all the work done in the project link:/projects/gsoc/2024/projects/automating-rpu-for-jenkinsci-organization/[Manage jenkinsci GitHub permissions as code] during Google Summer of Code 2024.
```

Reply:
Change already suggested previously

Reply:
Yes, but I also removed `https://www.jenkins.io` because it is handled by the `link` macro.